macros/EmbedInteractiveExample.ejs

```diff
@@ -0,0 +1,31 @@
+<%
+// Embeds a live sample from a mdn.github.io GitHub page
+//
+// Parameters:
+//  $0 - The URL of mdn.github.io page (relative)
+//  $1 - Keyword indicating iframe size: small, medium, or large
+//
+//  Example call {{EmbedInteractiveExample("interactive-examples/pages/css/animation.html", "small")}}
+//
+
+let url = "https://mdn.github.io/" + $0;
+let classesList = Array();
+
+// figure out classes
+if($0.includes('/js/')){
+    classesList.push('interactive-js');
+}
+
+if($1) {
+    classesList.push('interactive-' + $1);
+}
+
+let classes = '';
+for (var thisClass of classesList) {
+  classes += ' ' + thisClass;
+}
+
+let str = '<iframe class="interactive ' + classes + '" width="100%" height="250" frameborder="0" src="' + url + '"></iframe>';
```

Comment:
Yes, the CSS will override the height defined in the HTML.

The custom height will be discarded. We can't do custom heights because there is no way to make the CSS change the custom height for mobile pages. 

I didn't know there were other users of `EmbedGHLiveSample` :/ Should I leave that macro alone or should that not be using the macro?